lib/msal-node/src/cache/NodeCacheManager.ts

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+const fs = require('fs');
+
+export class NodeCacheManager {
```

Comment:
I think just `CacheManager` is fine, don't need `Node`.

Reply:
Done